services/movie_session.py

```diff
@@ -42,3 +42,9 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
+
+
+def get_taken_seats(movie_session_id: int) -> list:
+    tickets = (Ticket.objects.filter(movie_session=movie_session_id)
+               .values_list("row", "seat"))
+    return [{"row": ticket[0], "seat": ticket[1]} for ticket in tickets]
```

Comment:
why dont you just return <query>.values("row", "seat") instead of adding comprehension?